src/pretalx/schedule/forms.py

```diff
@@ -184,6 +184,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.resolution = "00:15:00"
         self.fields["name"].widget.attrs["placeholder"] = _("Room I")
+        self.fields["guid"].widget.attrs["placeholder"] = _(
```

Comment:
I don't think that's a very helpful placeholder, tbh – since this field should always come prefilled, no placeholder is probably better, in conjunction with a good help text.